package.json

```diff
@@ -27,18 +30,20 @@
   },
   "homepage": "https://github.com/ciena-frost/frost-standard#readme",
   "devDependencies": {
-    "chai": "^3.4.1",
-    "eslint": "^3.0.1",
+    "chai": "^4.0.0",
+    "eslint": "^4.1.0",
     "istanbul": "^0.4.2",
-    "mocha": "^2.4.2"
+    "mocha": "^3.4.2"
   },
   "dependencies": {
-    "babel-eslint": "^6.1.2",
-    "eslint-config-standard": "^5.3.5",
+    "babel-eslint": "^7.2.3",
+    "eslint-config-standard": "^10.2.1",
```

Comment:
Would prefer this to not be a float, as lots of new rules get introduced in major and minor versions (https://github.com/standard/standard/blob/master/CHANGELOG.md) that suddenly a codebase will fail on unexpectedly.  I'm assuming this is partially related to why you had to run the `import/first` rule off.